### PR TITLE
Support app parameters

### PR DESCRIPTION
**Pull request by rui1610**
This PR addresses the issue https://github.com/SAP-samples/btp-setup-automator/issues/313 so that required parameters for subscribing to apps are provided in the btpsa-usecase.json library.

### CODE REVIEW

File: config/templates/libs/BTPSA-USECASE.json [hunk 1]
```diff
@@ -236,11 +240,6 @@
                         "description": "name of the service instance",
                         "title": "name of the service instance"
                     },
-                    "labels":{
```
Comment (lechnerc77) [inline config/templates/libs/BTPSA-USECASE.json:239]: The functionality of  labels are removed with your commit - there seems to be something wrong with the rebase/merge

File: libs/btpsa-usecase.json [hunk 2]
```diff
@@ -226,11 +226,6 @@
                         "description": "name of the service instance",
                         "title": "name of the service instance"
                     },
-                    "labels":{
```
Comment (lechnerc77) [inline libs/btpsa-usecase.json:229]: The functionality of labels are removed with your commit - there seems to be something wrong with the rebase/merge

**Review by lechnerc77 — CHANGES_REQUESTED** [review]
Something went wrong with the rebase/merge of your branch towards the `dev` branch. The availability to define labels to resources would be removed by your commit. 
This needs to be corrected before merging the pull request

**Comment by rui1610** [comment]
Great catch! Don't know what happened. I've just fixed the issue and it should be fine now. Would you mind checking again?

**Review by lechnerc77 — APPROVED** [review]

**Pull request merged**